src/lib.rs

```diff
@@ -732,6 +783,24 @@ where
     }
 }
 
+impl<Data, DeSer> Database<Data, FileBackend, DeSer>
+where
+    Data: Serialize + DeserializeOwned + Clone + Send + Default,
+    DeSer: DeSerializer<Data> + Send + Sync + Clone,
+{
+    /// Load [`FileDatabase`] at `path` or initialise with `Data::default()`.
+    ///
+    /// Create new [`FileDatabase`] from the file at [`Path`](std::path::Path),
+    /// and load the contents. If the file does not exist, initialise with
+    /// `Data::default`.
+    pub fn load_from_path_or_default<S>(path: S) -> error::Result<Self>
+    where
+        S: AsRef<std::path::Path>,
+    {
+        Self::load_from_path_or(path, Data::default())
```

Comment:
This is technically not correct, I'd rather it uses `load_from_path_or_else(Data::default)` so that nothing gets allocated that is not used.

Reply:
Yes, you are right. I had use cases in mind where `Data::default()` is (almost) free. Then this (the current) implementation could be slightly faster, but for expensive `Data::default()` what you propose is much faster. I'll change it.

Reply:
Changed it and added tests that check for this behaviour (using a struct with `panic`ing `default`).